downstream/modules/platform/ref-controller-credential-virtualization.adoc

```diff
@@ -0,0 +1,27 @@
+[id="ref-controller-credential-virtualization"]
+
+= Red Hat Virtualization
```

Comment:
Should we be including this, given that RHV is being discontinued. (I think) ?

Reply:
Something for the SMEs to decide.